index.js

```diff
@@ -500,6 +500,17 @@ module.exports = {
           type: String,
           default: 'master',
           description: 'branch to push to'
+        }, {
+          name: 'push-condition-compare',
```

Comment:
It's not totally clear which of these is supposed to be coming from the CI system and which is configured ahead of time, can they be worded for more clarity?